modules/ROOT/pages/comments-commands-events-apis.adoc

```diff
@@ -10,3 +10,30 @@
 The Comments plugin provides the following {productname} commands.
 
 include::partial$commands/comments-cmds.adoc[]
+
+== API
+
+The Comments plugin exposes an API, `editor.plugins.tinycomments.getEventLog()`, which returns a log of all `CommentChange` events which are fired when:
+
+* A new comment is added (and who added it).
+* A comment is edited (and who edited it).
+* A reply to a comment is added (and who added the reply).
+* A comment is resolved (and who resolved the comment).
+* A comment is deleted (and who deleted the comment).
+
+:minimumversion: 6.1
+include::partial$misc/minimum-version.adoc[]
+
+The event log can be retrieved either in full or with the `after` option, which restricts the returned list to Comment events after the time-stamp date, both shown in the following:
+
+=== Example: Using the `+comments+` plugin APIs
+
+[source,js]
+----
+const comments = tinymce.activeEditor.plugins.tinycomments;
+
+console.log(comments.getEventLog());
+console.log(comments.getEventLog(
+    { after: '<iso-8601-format-time-stamp>' }
```

Comment:
I feel like we need an example timestamp here, or a better explanation of what the format is.

Reply:
I'll investigate, I pulled this from the release notes

Reply:
ah. there's so many comments on that I've lost track.

Reply:
[ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) is widely used and it’s very common as the format APIs take. So I guess I was assuming just the ISO number would ring bells for a fair proportion of the likely readership (it does for me, FWIW).

An alternative would be to put a generic version of the longest 8601 standard string as placeholder text:

`<yyyy-mm-ddThh:mm:ss+00:00>`

I’m not sure that’s more readable.

If we go with an example timestamp we should add a post-example note about reading the time-stamp and about how needs to be an ISO-8601 format string.

Something like:

```
[source,js]
----
console.log(comments.getEventLog());
console.log(comments.getEventLog(
  { after: '<2022-07-12T00:12:00+00:00>' }
)
----
```

The example above restricts the returned list to Comment events created after 12:00 UTC on 2022-07-12.

The entered time-stamp must be in ISO-8601 format.